languages/julia/main.go

```diff
@@ -37,10 +37,11 @@ func (p Julia) Eval(code string) {
 	C.eval(cstr)
 }
 
-func (p Julia) EvalFile(file string, args []string) {
+func (p Julia) EvalFile(file string, args []string) int {
 	cstr := C.CString(file)
-	C.eval_file(cstr)
+	code := C.eval_file(cstr)
```

Comment:
another alternative is to follow languages/lua/main.go:52 for consistency and defer the freeing of the `cstr`.